skills/csharp/experimental/phoneskill/Responses/Shared/PhoneSharedResponses.lg

```diff
@@ -1,54 +1,57 @@
+[import](..\Shared\Shared.lg)
+[import](PhoneSharedResponsesTexts.lg)
+
 # DidntUnderstandMessage(Data, Cards, Layout)
 [Activity
-    Text = @{DidntUnderstandMessage.Text(Data)}
-    Speak = @{DidntUnderstandMessage.Text(Data)}
-    Attachments = @{if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
-    AttachmentLayout = @{if(Layout == null, 'list', Layout)}
+    Text = ${DidntUnderstandMessage.Text(Data)}
+    Speak = ${DidntUnderstandMessage.Text(Data)}
+    Attachments = ${if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
+    AttachmentLayout = ${if(Layout == null, 'list', Layout)}
     InputHint = acceptingInput
 ]
 
 # CancellingMessage(Data, Cards, Layout)
 [Activity
-    Text = @{CancellingMessage.Text(Data)}
-    Speak = @{CancellingMessage.Text(Data)}
-    Attachments = @{if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
-    AttachmentLayout = @{if(Layout == null, 'list', Layout)}
+    Text = ${CancellingMessage.Text(Data)}
+    Speak = ${CancellingMessage.Text(Data)}
+    Attachments = ${if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
+    AttachmentLayout = ${if(Layout == null, 'list', Layout)}
     InputHint = acceptingInput
 ]
 
 # NoAuth(Data, Cards, Layout)
 [Activity
-    Text = @{NoAuth.Text(Data)}
-    Speak = @{NoAuth.Text(Data)}
-    Attachments = @{if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
-    AttachmentLayout = @{if(Layout == null, 'list', Layout)}
+    Text = ${NoAuth.Text(Data)}
+    Speak = ${NoAuth.Text(Data)}
+    Attachments = ${if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
+    AttachmentLayout = ${if(Layout == null, 'list', Layout)}
     InputHint = expectingInput
 ]
 
 # AuthFailed(Data, Cards, Layout)
 [Activity
-    Text = @{AuthFailed.Text(Data)}
-    Speak = @{AuthFailed.Text(Data)}
-    Attachments = @{if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
-    AttachmentLayout = @{if(Layout == null, 'list', Layout)}
+    Text = ${AuthFailed.Text(Data)}
+    Speak = ${AuthFailed.Text(Data)}
```

Comment:
AuthFailed have multiple texts

Reply:
LG should be able to just pick one from the list and evaluate right?